README.md

```diff
@@ -31,7 +31,7 @@ This script pulls the image and runs Theia IDE on http://localhost:3000 with the
     # Linux or macOS
     docker run -it --init -p 3000:3000 -v "$(pwd):/home/project:cached" theiaide/theia:next
     
-    # Windows
+    # Windows CMD
```

Comment:
`# Windows (cmd.exe)` ?

I think it would be fine to merge this change with your other PR: https://github.com/theia-ide/theia-apps/pull/339